java/component/keyple-calypso/src/main/java/org/eclipse/keyple/calypso/transaction/PoTransaction.java

```diff
@@ -1347,4 +1413,16 @@ public final void prepareDecreaseCounter(byte sfi, int counterNumber, int decVal
         poCommandManager.addRegularCommand(
                 new DecreaseCmdBuild(calypsoPo.getPoClass(), sfi, counterNumber, decValue));
     }
+
+    /**
+     * Builds a VerifyPin command without PIN presentation in order to get the attempt counter.<br>
+     * The PIN status will made available in CalypsoPo after the execution of process command.<br>
+     * Adds it to the list of commands to be sent with the next process command.
+     * 
+     * @see CalypsoPo {@code isPinBlocked and getPinAttemptRemaining methods}
```

Comment:
Bad use of @see. Use instead {@link with #} inside the comment block

Reply:
Done